src/Hl7.Fhir.ElementModel/IXHtml.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Hl7.Fhir.ElementModel
+{
+    public interface IXHtml
```

Comment:
We need another name here. This was just an example.

Reply:
Perhaps this will cover it: `ICcdaInfoSupplier`

Reply:
Sounds good to me. Renamed it.